scripts/zones/Upper_Jeuno/npcs/_6s1.lua

```diff
@@ -1,106 +1,124 @@
------------------------------------
+----------------------------------------------------------------------------------------------------------------------------
 -- Area: Upper Jeuno
 -- NPC:  Marble Bridge Eatery (Door)
 -- @pos -96.6 -0.2 92.3 244
------------------------------------
-package.loaded["scripts/zones/Upper_Jeuno/TextIDs"] = nil;
------------------------------------
-
-require("scripts/globals/settings");
-require("scripts/globals/missions");
-require("scripts/zones/Upper_Jeuno/TextIDs");
-
-local RajasRing = 15543;
-local SattvaRing = 15544;
-local TamasRing = 15545;
-
------------------------------------
+----------------------------------------------------------------------------------------------------------------------------
+package.loaded["scripts/zones/Upper_Jeuno/TextIDs"] = nil
+----------------------------------------------------------------------------------------------------------------------------
+require("scripts/zones/Upper_Jeuno/TextIDs")
+require("scripts/globals/missions")
+require("scripts/globals/settings")
+----------------------------------------------------------------------------------------------------------------------------
+local ring = {
```

Comment:
We're brace on newline ppl here. Its in the unfinished style guide. The one automatically linked to everyone the second the try to contribute. https://github.com/DarkstarProject/darkstar/blob/master/CONTRIBUTING.md